### PR TITLE
Add Dependabot

### PR DESCRIPTION
**Pull request by larouxn**
Add a [validated](https://dependabot.com/docs/config-file/validator/) Dependabot config so we can keep our Ruby and JavaScript dependencies fresh.
Screen Shot 2019-11-25 at 08 45 35

### CODE REVIEW

**Comment by larouxn** [comment]
Seems we already have Dependabot set up (https://github.com/chatwoot/chatwoot/pull/207) without a config. Likely don't need this PR.

**Comment by pranavrajs** [comment]
@larouxn Dependabot is already added, but I'd like to keep this open, I didn't know dependabot has a configuration. I would happy to take a look at this and see if it helps us to improve our workflow.

**Comment by larouxn** [comment]
👋 @pranavrajs, okay cool. Sounds good! There are a bunch of [configuration options](https://dependabot.com/docs/config-file/#available-configuration-options).

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: 1
+update_configs:
+  - package_manager: "ruby:bundler"
+    directory: "/"
+    update_schedule: "weekly"
+    default_reviewers:
+      - "who should go here?"
```
Comment (pranavrajs) [inline .dependabot/config.yml]: @larouxn This default reviewers can be 

1. sony-mathew
2. sojan-official
3. subintp

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    default_labels:
+      - "dependencies"
+    version_requirement_updates: "auto"
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "weekly"
+    default_reviewers:
+      - "who should go here?"
```
Comment (pranavrajs) [inline .dependabot/config.yml]: @larouxn Here default reviewers can be

pranavrajs
nithindavid

**Comment by pranavrajs** [comment]
@larouxn Could you please update the PR with the above mentioned changes.

Thanks

**Comment by larouxn** [comment]
👋 @pranavrajs, I have updated the config as outline above.

**Pull request merged**